src/components/NetworkTabs.tsx

```diff
@@ -0,0 +1,68 @@
+import { TabList } from "@mui/lab";
+import { NoSsr, Tab } from "@mui/material";
+import isEqual from "lodash/isEqual";
+import {
+  defaultStreamQueryOrdering,
+  defaultStreamQueryPaging,
+} from "../components/NetworkStreams";
+import { track } from "../hooks/useMatomo";
+import { useAppSelector } from "../redux/hooks";
+import { networks } from "../redux/networks";
+import { sfSubgraph } from "../redux/store";
+import NetworkDisplay from "./NetworkDisplay";
+
+type NetworkTabsProps = {
+  activeTab: string;
+  setActiveTab: (...args: any[]) => void;
+};
+
+const NetworkTabs: React.FC<NetworkTabsProps> = ({ setActiveTab }) => {
+  const prefetchStreamsQuery = sfSubgraph.usePrefetch("streams", {
+    ifOlderThan: 45,
+  });
+
+  const displayedTestnetChainIds = useAppSelector(
+    (state) =>
+      Object.entries(state.appPreferences.displayedTestNets)
+        .filter(([_, isDisplayed]) => isDisplayed)
+        .map(([chainId]) => Number(chainId)),
+    isEqual
+  );
+
+  return (
+    <NoSsr>
+      <TabList
+        variant="scrollable"
+        scrollButtons="auto"
+        data-cy={"landing-page-networks"}
```

Comment:
This was the same across all pages, I didn't want to change it and break e2e tests if there are some, but this should be adjusted, to which page this is opened right?

Reply:
@tokdaniel The data-cy was probobly created before we had token and protocol pages , ill re-name it , no need to have it per-page as its just the parent of all the network buttons and those have the data-cy with the slug name in them per network